code/api/src/migrations/20210208174208-add_stylePreference_to_users.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.addColumn(
+      'users',
+      'stylePreference',
```

Comment:
stylePreference used for users, while styleType is used for subscriptions and products.